modules/builds-displaying-webhook-urls.adoc

```diff
@@ -5,7 +5,7 @@
 [id="builds-displaying-webhook-urls_{context}"]
 = Displaying webhook URLs
 
-You can use the following command to display webhook URLs associated with a `BuildConfig`. If the command does not display any webhook URLs, then no webhook trigger is defined for that build configuration. See Setting Triggers to manually add triggers.
+You can use the following command to display webhook URLs associated with a build configuration. If the command does not display any webhook URLs, then no webhook trigger is defined for that build configuration. See setting triggers to manually add triggers.
```

Comment:
Is there supposed to be a link here? Or an additional resource section w/link following this module to take the user to this topic?

Reply:
Thank you for pointing it out. I'm just going to remove that line. There was an intention to put a link there someday, but it builds in the same assembly so it's not exactly necessary.